Physics/CdrPlasma/PlasmaModels/CdrPlasmaJSON/CD_CdrPlasmaJSON.H

```diff
@@ -647,7 +649,46 @@ namespace Physics {
       /*!
 	@brief List of dielectric reactions
       */
-      std::vector<CdrPlasmaSurfaceReactionJSON> m_dielectricReactions;      
+      std::vector<CdrPlasmaSurfaceReactionJSON> m_dielectricReactions;
+
+       // ===========================================
+      // DOMAIN SURFACE REACTIONS BEGIN HERE
+      // ===========================================
+
+      /*!
+	@brief Lookup method for the domain reaction ratesThe pair is made up of an int representing direction (0=x, 1=y, 2=z) and a Side::LoHiSide
+	representing side (Side::Lo, Side::Hi)
+      */
+      std::map<std::pair<int, Side::LoHiSide>, std::map<int, LookupMethod>> m_domainReactionLookup;
+
+      /*!
+	@brief Constant domain reaction rate. The pair is made up of an int representing direction (0=x, 1=y, 2=z) and a Side::LoHiSide
+	representing side (Side::Lo, Side::Hi)
+      */
+      std::map<std::pair<int, Side::LoHiSide>, std::map<int, Real>> m_domainReactionConstants;
+
+      /*!
+	@brief Domain reaction effiencies. Used for scaling reactions on domains in a "generic" way. 
+	The pair is made up of an int representing direction (0=x, 1=y, 2=z) and a Side::LoHiSide
+	representing side (Side::Lo, Side::Hi)
+      */
+      std::map<std::pair<int, Side::LoHiSide>, std::map<int, FunctionEX>> m_domainReactionEfficiencies;
+
+      /*! 
+	@brief List of domain reactions. The pair is made up of an int representing direction (0=x, 1=y, 2=z) and a Side::LoHiSide
+	representing side (Side::Lo, Side::Hi)
+      */
+      std::map<std::pair<int, Side::LoHiSide>, std::vector<CdrPlasmaSurfaceReactionJSON>> m_domainReactions;
+
+      /*! 
+	@brief map to translate dir from char to int
+      */
+      const std::map<char, int> m_dirCharToInt{{'x', 0}, {'y', 1}, {'z', 2}};
```

Comment:
Does this have to be a member? If yes, make it constexpr?

Reply:
initially I made it as a local variable in the CdrPlasmaJSON::parseDomainReactions()-function, but I changed it to a member when I needed it in both parseDomainReactionRate() and parseDomainReactionScaling() as well. The alternative would be to have it as a variable in all three functions, which option do you prefer?

And yes, constexpr is a good idea if it stays as a member

Reply:
Used more than once -> member.

Reply:
Do you want me to place it somewhere else in the .h-file instead, or is it ok to keep it as is?

Reply:
It seems like std::map cannot be constexpr, only const

Reply:
Ok, keep as is.